documentation/src/cli.md

```diff
@@ -56,38 +68,62 @@ Filecoin wallets are stored under the Forest data directory (e.g.,
 
 All wallet commands require write permissions and an admin token (`--token`) to
 interact with the keystore. The admin token can be retrieved from forest startup
-logs or by running it with `--save-token <PATH>`.
+logs or by including the flag `--save-token <PATH>` during `forest` invocation.
+
+### Balance:
+
+Retrieve the FIL balance of a given address. Usage:
+`forest-cli --token <admin_token> wallet balance <address>`
+
+### Default:
+
+Get the default, persisted address from the keystore. Usage:
+`forest-cli --token <admin_token> wallet default`
+
+### Has:
 
-Balance Retrieve the FIL balance of a given address Usage:
-`forest-cli wallet balance <address>`
+Check if an address exists in the keystore. Outputs `true` if address exists and
+`false` if address does not exist. Usage:
+`forest-cli --token <admin_token> wallet has <address>`
 
-Default Get the default, persisted address from the keystore Usage:
-`forest-cli wallet default`
+### List:
 
-Has Check if an address exists in the keystore shows true/false if exists or
-doesn't Usage: `forest-cli wallet has <address>`
+Display the keys in the keystore. Usage:
+`forest-cli --token <admin_token> wallet list`
 
-List Display the keys in the keystore Usage: `forest-cli wallet list`
+### New:
 
-New Create a new wallet The signature type can either be secp256k1 or bls.
-Defaults to use bls Usage: `forest-cli wallet new <bls/secp256k1>`
+Create a new wallet. The signature type can either be secp256k1 or bls. Defaults
+to use secp256k1. Usage:
+`forest-cli --token <admin_token> wallet new <bls/secp256k1>`
```

Comment:
```suggestion
`forest-cli --token <admin_token> wallet new [ bls | secp256k1 ]`
```

square brackets are idiomatic for optional parameters (as angle brackets are for mandatory ones)